test/helpers/governance.js

```diff
@@ -62,10 +62,10 @@ class GovernorHelper {
         );
   }
 
-  cancel(opts = null) {
+  cancel(usePublic = false, opts = null) {
```

Comment:
Using the internal function instead of the external function can lead to a very wrong test. So I would default to `usePublic = true` so that the test fails instead of silently passing. To improve readability I would also change this from a boolean to a string `'internal' | 'external'` (or `public`).

```suggestion
  cancel(visibility = 'external', opts = null) {
```

Reply:
done